.github/workflows/ci.yml

```diff
@@ -64,5 +64,12 @@ jobs:
         with:
           command: test
 
+      - name: cargo test -p accesskit_windows
+        if: matrix.os == 'windows-2019'
+        uses: actions-rs/cargo@v1
```

Comment:
Actually, could you pass `—workspace` to the existing test run instead

Reply:
I don't think that works because it also runs the windows tests on non-windows platforms and mac tests on the windows platform. 

At least when I run it locally I get this error
```
   --> C:\Users\Paul\.cargo\registry\src\index.crates.io-6f17d22bba15001f\objc2-0.3.0-beta.3.patch-leaks.2\src\foundation\mod.rs:102:36
    |
102 | #[link(name = "Foundation", kind = "framework")]
For more information about this error, try `rustc --explain E0455`.
error: could not compile `objc2` (lib) due to previous error
```
"Foundation" sounds like an Apple thing I think.

Reply:
I can try it though if you want